src/components/gallery/GalleryCards.jsx

```diff
@@ -25,36 +46,44 @@ const GalleryCards = ({ cards, title }) => {
   };
 
   return (
-    <div className=" flex justify-center flex-col items-center w-full gap-6 mb-3">
-      <GalleryHeader text={title} />
-      <div className="flex w-full justify-between mt-2">
-        <button
-          onClick={handleLeftClick}
-          className="flex justify-center items-center"
-        >
-          <FaAngleLeft className="md:text-8xl text-6xl text-art-pink-400 hover:text-art-purple-300" />
-        </button>
-        <Image
-          src={cards[currentCardIndex].image}
-          alt="Image"
-          className="md:w-80 md:h-80 w-60 h-60 rounded-xl flex justify-center items-center bg-art-purple-100"
-        />
-        <Image
-          src={cards[currentCardIndex + 1].image}
-          alt="Image"
-          className="ml-4 w-80 h-80 rounded-xl justify-center items-center bg-art-purple-100 hidden md:block"
-        />
-        <Image
-          src={cards[currentCardIndex + 2].image}
-          alt="Image"
-          className="ml-4 w-80 h-80 rounded-xl justify-center items-center bg-art-purple-100 hidden lg:block"
-        />
-        <button
-          onClick={handleRightClick}
-          className="flex justify-center items-center"
-        >
-          <FaAngleRight className="md:text-8xl text-6xl text-art-pink-400 hover:text-art-purple-300" />
-        </button>
+    <div className="flex justify-center flex-col items-center w-full gap-6 mb-3">
+      <div
+        className={`${
+          headerVisible
+            ? "translate-y-0 opacity-100"
+            : "-translate-y-10 opacity-0"
+        } transition-all duration-700 ease-in-out`}
```

Comment:
pull from dev or copy paste the exisiting code from dev since your code doesn't look the same from the dev branch